frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/validation/PhoneNumberCodeProcessor.java

```diff
@@ -144,7 +145,13 @@ public void processSuccessfulCodeRequest(String ipAddress, String persistentSess
             String phoneNumber =
                     PhoneNumberHelper.formatPhoneNumber(codeRequest.getProfileInformation());
 
-            submitRequestToExperianPhoneCheckSQSQueue(journeyType, phoneNumber);
+            if (!isValidTestNumberForEnvironment(phoneNumber)) {
```

Comment:
Minor, I'd swap this condition round so it's 
```
if (isValidTestNumberForEnvironment(phoneNumber)) {
    ...don't submit to experian 
} else {
    ...do the stuff to submit to experian 
}

```
as it's a bit harder to parse nots and we don't have to here